app/assets/stylesheets/common/admin/customize.scss

```diff
@@ -168,6 +168,25 @@
     .control-unit {
       margin-top: 0.5em;
       margin-bottom: 2em;
+
+      .translation-selector-container {
+        display: grid;
+        grid-template-columns: calc(50% - 2px) 30% 20%;
+
+        @media screen and (max-width: 700px) {
+          grid-template-columns: none;
+        }
+
+        @media screen and (min-width: 700px) and (max-width: 780px) {
+          grid-template-columns: calc(50% - 4.5vw) 30% 20%;
```

Comment:
>Could we use a flex container here and make the dropdown right-aligned via flex spacing and margin-left: auto? Gotta keep in mind as well that the title (Theme Translations) can be longer in some more verbose languages.

When using this setup, on wider screens due to the structure of this section, the row looks like this:


https://github.com/discourse/discourse/assets/70247653/9deced8d-6615-401f-a769-2c022d8edf7f


But while recording the video, I've thought of a way that seems better on wider screens using a defined width and on smaller ones, using the 100% as usual. You can see a video of the solution below; I've added a media query in 700px - 768px because of how the UI changes in this section:


https://github.com/discourse/discourse/assets/70247653/df7e60eb-c8e3-4a1d-9d4b-0dcacdf57557